src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -1793,6 +1793,16 @@ internal static void ApplyApplicationInsightsConfig(JObject configJson, ScriptHo
                         }
                     }
                 }
+
+                if (configSection.TryGetValue("endpointAddress", out value))
+                {
+                    scriptConfig.ApplicationInsightsEndpointAddress = (string)value;
+                }
+
+                if (configSection.TryGetValue("quickPulseServiceEndpoint", out value))
```

Comment:
Let's line these with the App Insights Connection Strings, which use `IngestionEndpoint` and `LiveEndpoint`: https://docs.microsoft.com/en-us/azure/azure-monitor/app/sdk-connection-string?tabs=net#syntax

Probably need to rename back in SDK as well.